sites/all/themes/cetaviano/src-js/script.js

```diff
@@ -45,8 +45,10 @@
         });
       }
 
+      $("#edit-search-api-multi-fulltext").val("");
+
       function requiredForm() {
-        $(".entitytype-formulario_envio_de_pago-form input").attr("required","required");
+        $("#edit-search-api-multi-fulltext").add(".entitytype-formulario_envio_de_pago-form input").attr("required","required");
         $("#edit-field-form-email-und-0-email").attr("type", "email");
```

Comment:
Si, pero decís que hay un issue, el cual no pude revisar a fondo, así que
no lo aprobé, además de que no entiendo bien esa validación

El form de drupal debería validar cuando se hace submit, para esta versión
que se va a lanzar podemos dejarlo solo con la validación de drupal y
dejamos como un nice to have la validación que queres mwter

On Wednesday, 20 January 2016, fmorar <notifications@github.com> wrote:

> In sites/all/themes/cetaviano/src-js/script.js
> <https://github.com/cetav-ddw/cetav-2015/pull/52#discussion_r50348682>:
>
> >        function requiredForm() {
> > -        $(".entitytype-formulario_envio_de_pago-form input").attr("required","required");
> > +        $("#edit-search-api-multi-fulltext").add(".entitytype-formulario_envio_de_pago-form input").attr("required","required");
> >          $("#edit-field-form-email-und-0-email").attr("type", "email");
>
> Jose vos viste el pull del otro js que te mande ? El ene 20, 2016 7:21 PM,
> "Jose" <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> escribió:
> … <#-351778091_>
> In sites/all/themes/cetaviano/src-js/script.js <#52 (comment)
> <https://github.com/cetav-ddw/cetav-2015/pull/52#discussion_r50348300>>:
> > function requiredForm() { > -
> $(".entitytype-formulario_envio_de_pago-form
> input").attr("required","required"); > +
> $("#edit-search-api-multi-fulltext").add(".entitytype-formulario_envio_de_pago-form
> input").attr("required","required"); >
> $("#edit-field-form-email-und-0-email").attr("type", "email"); Que hace el
> add? — Reply to this email directly or view it on GitHub <
> https://github.com/cetav-ddw/cetav-2015/pull/52/files#r50348300>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cetav-ddw/cetav-2015/pull/52/files#r50348682>.
>


-- 
-- José
@leivajd